sorc/rtma_post.fd/sorc/ncep_post.fd/MISCLN.f

```diff
@@ -4079,7 +4079,7 @@ SUBROUTINE MISCLN
                DO I=ISTA,IEND
                  IF(LLOW(I,J)<spval.and.LUPP(I,J)<spval.and.&
                  VH(I,J,MIDCAL)<spval.and.VH(I,J,LLOW(I,J))<spval)THEN
-                       MIDCAL=INT(LLOW(I,J)+D50*(IEQL(I,J)-LLOW(I,J)))
+                       MIDCAL=INT(LLOW(I,J)+D50*(LUPP(I,J)-LLOW(I,J)))
```

Comment:
Hi @EdwardColon-NOAA,

Same question as I asked for code CALHEL3.f above. Is it a fix to the bug of the weird thing with wind shear?

Thank you!

-Gang